pootle/apps/pootle_fs/forms.py

```diff
@@ -21,18 +21,22 @@
 class ProjectFSAdminForm(forms.Form):
 
     fs_type = forms.ChoiceField(
+        # Translators: Pootle FS backend to deal with translations.
         label=_("Filesystem backend"),
-        help_text=_("Select a backend filesystem"),
+        help_text=_("Select a filesystem backend"),
         choices=(),
         widget=forms.Select(
             attrs={'class': 'js-select2'}))
     fs_url = forms.CharField(
+        # Translators: This tells where to find the translations.
         label=_("Backend URL or path"),
         help_text=_(
             "The URL or path to your translation files"))
     translation_path = forms.CharField(
+        # Translators: Expression reflecting how the filesystem looks like.
```

Comment:
Still think this shouldn't be in a doc, since now you can't point to the explanation :(

Expression that maps the location of the translation files so that Pootle FS can know what is translatable and where they are located.